test/src/main/java/org/apache/accumulo/test/compaction/CompactionPriorityQueueMetricsIT.java

```diff
@@ -97,7 +98,7 @@ public class CompactionPriorityQueueMetricsIT extends SharedMiniClusterBase {
   private String rootPath;
 
   public static final String QUEUE1 = "METRICSQ1";
-  public static final String QUEUE1_METRIC_LABEL = "e." + MetricsUtil.formatString(QUEUE1);
+  public static final String QUEUE1_METRIC_LABEL = QUEUE1;
```

Comment:
@dlmarion  and @ddanielr I am uncertain about this change.  I understand why I needed to remove `e.` for this test.  However I am not sure why I needed to remove the call to `MetricsUtil.formatString()`.  I determined I needed to remove that to make the test pass by printing things out, but not sure if this is the correct thing to do.

Wondering if its an indication of metrics names not being formated as expected, but not sure.  Going to look into this more later.  If either of you know anything about this please let me know.

Reply:
I'll look into this.

The whole point of the format call was to make sure that the metric labels are syntactically correct for micrometer.

Reply:
@ddanielr I noticed ExternalCompactionMetricsIT was failing and made the following changes locally to get it to pass. So it seems to have a similar issue.

```diff
diff --git a/test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionMetricsIT.java b/test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionMetricsIT.java
index 1dde2045c9..7d56a0b012 100644
--- a/test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionMetricsIT.java
+++ b/test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionMetricsIT.java
@@ -152,8 +152,11 @@ public class ExternalCompactionMetricsIT extends SharedMiniClusterBase {
       // wait until expected number of queued are seen in metrics
       while (!sawDCQ1_5 || !sawDCQ2_10) {
         Metric qm = queueMetrics.take();
-        sawDCQ1_5 |= match(qm, "dcq1", "5");
-        sawDCQ2_10 |= match(qm, "dcq2", "10");
+        if(qm.toString().contains("DCQ")) {
+          System.out.println("qm1:" + qm);
+        }
+        sawDCQ1_5 |= match(qm, "DCQ1", "5");
+        sawDCQ2_10 |= match(qm, "DCQ2", "10");
       }
 
       getCluster().getConfig().getClusterServerConfiguration().addCompactorResourceGroup(GROUP1, 1);
@@ -166,8 +169,11 @@ public class ExternalCompactionMetricsIT extends SharedMiniClusterBase {
       // wait until queued goes to zero in metrics
       while (!sawDCQ1_0 || !sawDCQ2_0) {
         Metric qm = queueMetrics.take();
-        sawDCQ1_0 |= match(qm, "dcq1", "0");
-        sawDCQ2_0 |= match(qm, "dcq2", "0");
+        if(qm.toString().contains("DCQ")) {
+          System.out.println("qm2:" + qm);
+        }
+        sawDCQ1_0 |= match(qm, "DCQ1", "0");
+        sawDCQ2_0 |= match(qm, "DCQ2", "0");
       }
 
       shutdownTailer.set(true);
@@ -194,7 +200,7 @@ public class ExternalCompactionMetricsIT extends SharedMiniClusterBase {
   private static boolean match(Metric input, String queue, String value) {
     if (input.getTags() != null) {
       String id = input.getTags().get("queue.id");
-      if (id != null && id.equals("e." + queue) && input.getValue().equals(value)) {
+      if (id != null && id.equals(queue) && input.getValue().equals(value)) {
         return true;
       }
     }

```